lib/memcached.rb

```diff
@@ -21,12 +21,17 @@ module Rlibmemcached
 class Memcached
   Lib = Rlibmemcached
   raise "libmemcached 0.32 required; you somehow linked to #{Lib.memcached_lib_version}." unless "0.32" == Lib.memcached_lib_version
+  private_constant :Lib
+
   VERSION = File.read("#{File.dirname(__FILE__)}/../CHANGELOG")[/v([\d\.]+)\./, 1]
 
   autoload :Rails, 'memcached/rails'
+  deprecate_constant :Rails
```

Comment:
👋 @dylanahsmith since this is now deprecated; do you know what is going to be replacing this?

Reply:
`Memcached.new` can be used directly instead.

If you are using this with rails, then you will need a cache adapter anyways, which the memcached_store gem provides.  [memcached_store gem doesn't support Memcached::Rails](https://github.com/Shopify/memcached_store/blob/v2.3.1/lib/active_support/cache/memcached_store.rb#L74) and I don't think it has for a long time.